clang/include/clang/Sema/Sema.h

```diff
@@ -13372,6 +13376,17 @@ class Sema final : public SemaBase {
       bool SkipForSpecialization = false,
       bool ForDefaultArgumentSubstitution = false);
 
+  /// Apart from storing the result to \p Result, this behaves the same as
+  /// another overload.
```

Comment:
It would be cleaner to put the documentation on this overload, declare it first, and then say that the other overload creates a new MultiLevelTemplateArgumentList and call getTemplateInstantiationArgs with it as the first param)